packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -111,32 +119,45 @@ const isEmpty = computed(() => props.options.length === 0);
       <OnyxEmpty>{{ t("empty") }}</OnyxEmpty>
     </slot>
 
-    <ul v-else v-scroll-end v-bind="listbox" class="onyx-listbox__options">
-      <OnyxListboxOption
-        v-for="option in props.options"
-        :key="option.id.toString()"
+    <div v-else v-scroll-end v-bind="listbox" class="onyx-listbox__wrapper">
+      <ul
+        v-for="([group, options], index) in Object.entries(groupedOptions)"
+        :key="index"
+        class="onyx-listbox__group"
         v-bind="
-          headlessOption({
-            value: option.id,
-            label: option.label,
-            disabled: option.disabled,
-            selected: option.id === props.modelValue,
+          headlessGroup({
+            label: group,
           })
         "
-        :active="option.id === activeOption"
       >
-        {{ option.label }}
-      </OnyxListboxOption>
-
-      <li v-if="props.lazyLoading?.loading" class="onyx-listbox__slot">
-        <OnyxLoadingIndicator class="onyx-listbox__loading" />
-      </li>
-
-      <li v-if="slots.optionsEnd" class="onyx-listbox__slot">
-        <slot name="optionsEnd"></slot>
-      </li>
-    </ul>
-
+        <span v-if="group != ''" class="onyx-listbox__group-name onyx-text--small">{{
```

Comment:
I am not sure if <span> should be used inside <ul>. Maybe we should change it to also be <li>?

Reply:
Changing it can be tricky because of the ARIA expectations of the hierarchy. I mentioned this in our meeting earlier.

Reply:
But wasn't this only caused because of the `v-bind="headlessOption"`?
When using just a `<li>` for the group label without binding the headless option should be fine I guess.
Can we try it out?

Reply:
It works with role="presentation" :)